experimental/ExtendTeamsforM365/client/pages/productDetail.html

```diff
@@ -17,9 +35,9 @@
     <div id="content">
         <p>Loading...</p>
     </div>
-    <table id="orders">
+    <table id="orders" style="overflow: auto;" style="overflow: auto;">
```

Comment:
move to css file, other places as well.

Reply:
Changed and shifted to northwind-teams.css